### PR TITLE
Removing trailing whitespace using the shell script

### PR DESCRIPTION
**Pull request by DafinaTrufas**

### CODE REVIEW

File: scripts/remove-whitespace.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+find . -iname "*.v" -exec sed -i -e "s/[[:blank:]]*$//" "{}" \;
```
Comment (wkolowski) [inline scripts/remove-whitespace.sh:2]: Please add a newline at the end.

**Review by wkolowski — APPROVED** [review]
Looks ok, but I didn't triple-check precisely how the regex works.

File: scripts/remove-whitespace.sh (hunk 1 above)
Comment (DafinaTrufas) [reply scripts/remove-whitespace.sh]: Yes, indeed, I will open a seaparate PR for the newline-deduplication task.

**Review by palmskog — APPROVED** [review]
Looks good to me.

**Pull request merged**